timesketch/frontend-ng/src/components/Explore/EventDetail.vue

```diff
@@ -228,18 +237,33 @@ limitations under the License.
         </v-card>
       </v-col>
     </v-row>
+    <v-dialog
+      v-model="aggregatorDialog"
+      @click:outside="$event => this.aggregatorDialog = false"  
+    >
+      <ts-aggregate-dialog
+        :eventKey="eventKey"
+        :eventValue="eventValue"
+        :eventTimestamp="eventTimestamp"
+        :eventTimestampDesc="eventTimestampDesc"
+        :sketchId="this.sketch.id"
```

Comment:
You don't need this. You can use a sketch computed property in the dialog component instead.

Reply:
Done